src/rax_malloc.h

```diff
@@ -40,5 +40,5 @@
 #include "zmalloc.h"
 #define rax_malloc(size) zmalloc(size, MALLOC_SHARED)
 #define rax_realloc(ptr, size) zrealloc(ptr, size, MALLOC_SHARED)
-#define rax_free zfree
+#define rax_free(ptr) zfree((ptr))
```

Comment:
Only this is relevant to the first commit.

Reply:
The old way would still work but this is more clear for sure. The preprocessor just does a simple replace of text so the brackets in the caller will get left and become the brackets for the function call.